src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_diagnostics_templates.py

```diff
@@ -278,248 +278,702 @@ def get_default_diag_config(is_windows):
     else:
         return {
             "ladCfg": {
```

Comment:
This is a recorded test. Once you consolidate the test scenario, just delete the [recording](https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/tests/recordings/test_diagnostics_extension_install.yaml) and rerun the test to produce new recordings. Let me know if you need more information